src/tests/functional/plugin/cpu/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

```diff
@@ -81,10 +81,17 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::Values("MULTI", "AUTO"));
 
 const std::vector<ov::AnyMap> multiConfigs = {
-        {ov::device::priorities(CommonTestUtils::DEVICE_CPU)}};
+        {ov::device::priorities(CommonTestUtils::DEVICE_CPU)},
+        {ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT)},
```

Comment:
will this load GPU in auto?

Reply:
Will update to load to CPU only in Auto.